### PR TITLE
doc(release): prepare release notes for modules 23.04

### PR DESCRIPTION
**Pull request by tuntoja**
## Description

Prepare release notes for:
* ANOMALY DETECTION 23.04.2
* AUTODISCOVERY 23.04.1
* BAM 23.04.2
* LM 23.04.1
* PPM 23.04.1
* MAP 23.04.1
* MBI 23.04.1

## Target version (i.e. version that this PR changes)

- [ ] 21.10.x (staging)
- [ ] 22.04.x (staging)
- [ ] 22.10.x (staging)
- [x] 23.04.x (staging)
- [ ] Cloud (staging)
- [ ] Monitoring Connectors (staging)
- [ ] 23.10.x (next)

### CODE REVIEW

**Review by nmagniezCentreon — APPROVED** [review]

File: i18n/fr/docusaurus-plugin-content-docs/version-23.04/releases/centreon-commercial-extensions.md [hunk 1]
```diff
@@ -17,6 +17,38 @@ Retrouvez plus de détails sur la version 23.04 dans notre [post de blog](https:
...
+- [Licensing] Improved automatic license renewal.
+- [UI] Make metric link modal labels more explicit by replacing "metric 1" and "metric 2" by "metric in" and "metric out".
+- [UI] You can now hide the label of a resource.
+- [UX] You can now duplicate maps.
+- [Viewer] You can now define a label for a URL shape.
+- [Widget] Fixed incorrect behavior when saving the MAP widget.
+
+#### Bug fixes
+
+- [Editor] Fixed a bug where empty metric names were blocking edition.
```
Comment (nmagniezCentreon) [inline i18n/fr/docusaurus-plugin-content-docs/version-23.04/releases/centreon-commercial-extensions.md:35]: replace with [Editor] Fixed a bug where empty metric min or max prevent to save a map.
(sorry for the late change)

**Review by nmagniezCentreon — CHANGES_REQUESTED** [review]

File: i18n/fr/docusaurus-plugin-content-docs/version-23.04/releases/centreon-commercial-extensions.md (hunk 1 above)
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs/version-23.04/releases/centreon-commercial-extensions.md:35]:
```suggestion
- [Editor] Fixed a bug where empty metric min or max prevented users from saving a map.
```

File: versioned_docs/version-23.04/releases/centreon-commercial-extensions.md [hunk 2]
```diff
@@ -18,6 +18,38 @@ Read more about version 23.04 in our [blog post](https://www.centreon.com/centre
...
+- [Licensing] Improved automatic license renewal.
+- [UI] Make metric link modal labels more explicit by replacing "metric 1" and "metric 2" by "metric in" and "metric out".
+- [UI] You can now hide the label of a resource.
+- [UX] You can now duplicate maps.
+- [Viewer] You can now define a label for a URL shape.
+- [Widget] Fixed incorrect behavior when saving the MAP widget.
+
+#### Bug fixes
+
+- [Editor] Fixed a bug where empty metric names were blocking edition.
```
Comment (cg-tw) [inline versioned_docs/version-23.04/releases/centreon-commercial-extensions.md:36]:
```suggestion
- [Editor] Fixed a bug where empty metric min or max prevented to save a map.
```

File: i18n/fr/docusaurus-plugin-content-docs/version-23.04/releases/centreon-commercial-extensions.md [hunk 3]
```diff
@@ -17,6 +17,38 @@ Retrouvez plus de détails sur la version 23.04 dans notre [post de blog](https:
...
+- [Editor] Fixed metric links when no metric name is filled in.
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs/version-23.04/releases/centreon-commercial-extensions.md:36]:
```suggestion
- [Editor] Fixed metric links when no metric name was filled in.
```

File: i18n/fr/docusaurus-plugin-content-docs/version-23.04/releases/centreon-commercial-extensions.md [hunk 4]
```diff
@@ -17,6 +17,38 @@ Retrouvez plus de détails sur la version 23.04 dans notre [post de blog](https:
...
+- [Server] Enable to parse metric from Broker NEB packets.
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs/version-23.04/releases/centreon-commercial-extensions.md:37]:
```suggestion
- [Server] Made it possible to parse metrics from Broker NEB packets.
```

File: i18n/fr/docusaurus-plugin-content-docs/version-23.04/releases/centreon-commercial-extensions.md [hunk 5]
```diff
@@ -17,6 +17,38 @@ Retrouvez plus de détails sur la version 23.04 dans notre [post de blog](https:
...
+- [Server] Fixed a critical error when editing a Geo view.
+- [Server] Fixed an issue causing MAP server to crash at start up due to service group loading.
+- [Server] Fixed an issue preventing to save a map with "not a diagram" error message.
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs/version-23.04/releases/centreon-commercial-extensions.md:40]:
```suggestion
- [Server] Fixed an issue preventing users from saving a map with a "not a diagram" error message.
```

File: i18n/fr/docusaurus-plugin-content-docs/version-23.04/releases/centreon-commercial-extensions.md [hunk 6]
```diff
@@ -17,6 +17,38 @@ Retrouvez plus de détails sur la version 23.04 dans notre [post de blog](https:
...
+- [Server] Fixed an issue which made diagnostic to raise an error if a custom URI is used.
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs/version-23.04/releases/centreon-commercial-extensions.md:41]:
```suggestion
- [Server] Fixed an issue which made diagnostic.sh raise an error if a custom URI was used.
```

File: i18n/fr/docusaurus-plugin-content-docs/version-23.04/releases/centreon-commercial-extensions.md [hunk 7]
```diff
@@ -17,6 +17,38 @@ Retrouvez plus de détails sur la version 23.04 dans notre [post de blog](https:
...
+- [UI] Fixed cosmetic issues on breadcrumb chips.
+- [UI] Fixed the selection of layers in Geo views.
+- [UI] Fixed view freezes beyond 500 elements.
+- [Viewer] Fixed a regression causing output not to be displayed in widget output.
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs/version-23.04/releases/centreon-commercial-extensions.md:45]:
```suggestion
- [Viewer] Fixed a regression causing output not to be displayed in the output widget.
```

File: i18n/fr/docusaurus-plugin-content-docs/version-23.04/releases/centreon-commercial-extensions.md [hunk 8]
```diff
@@ -17,6 +17,38 @@ Retrouvez plus de détails sur la version 23.04 dans notre [post de blog](https:
...
+- [Viewer] Fixed an issue making links not to be reactivated when the associated resource goes back on line.
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs/version-23.04/releases/centreon-commercial-extensions.md:46]:
```suggestion
- [Viewer] Fixed an issue that caused links not to be reactivated when the associated resource went back online.
```

File: i18n/fr/docusaurus-plugin-content-docs/version-23.04/releases/centreon-commercial-extensions.md [hunk 9]
```diff
@@ -25,6 +57,20 @@ Release date: `April 26, 2023`
 
 ## Centreon BAM
 
+### 23.04.2
+
+Release date: `July 20, 2023`
+
+#### Enhancements
+
+- [Rules] Fixed boolean rules when resource contains boolean operator in its name.
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs/version-23.04/releases/centreon-commercial-extensions.md:66]:
```suggestion
- [Rules] Fixed boolean rules when resource contained a boolean operator in its name.
```

File: i18n/fr/docusaurus-plugin-content-docs/version-23.04/releases/centreon-commercial-extensions.md [hunk 10]
```diff
@@ -25,6 +57,20 @@ Release date: `April 26, 2023`
...
+
+#### Bug fixes
+
+- [Rules] Fixed boolean rules when resource contains boolean operator in its name.
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs/version-23.04/releases/centreon-commercial-extensions.md:70]: déjà dans enhancements

File: i18n/fr/docusaurus-plugin-content-docs/version-23.04/releases/centreon-commercial-extensions.md [hunk 11]
```diff
@@ -25,6 +57,20 @@ Release date: `April 26, 2023`
...
+- [UI] Fixed the column name display in the acknowledgment description pop-up.
+- Fixed retrieving list of BAs for users with ACL.
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs/version-23.04/releases/centreon-commercial-extensions.md:72]:
```suggestion
- Fixed how lists of BAs were retrieved for users with ACLs.
```

File: versioned_docs/version-23.04/releases/centreon-commercial-extensions.md [hunk 12]
```diff
@@ -18,6 +18,38 @@ Read more about version 23.04 in our [blog post](https://www.centreon.com/centre
...
 ## Centreon MAP
 
+### 23.04.1
+
+Release date: `July 20, 2023`
+
+#### Enhancements
+
+- [Licensing] Improved automatic license renewal.
+- [UI] Make metric link modal labels more explicit by replacing "metric 1" and "metric 2" by "metric in" and "metric out".
```
Comment (cg-tw) [inline versioned_docs/version-23.04/releases/centreon-commercial-extensions.md:28]:
```suggestion
- [UI] Made metric link modal labels more explicit by replacing "metric 1" and "metric 2" by "metric in" and "metric out".
```

File: versioned_docs/version-23.04/releases/centreon-commercial-extensions.md [hunk 13]
```diff
@@ -18,6 +18,38 @@ Read more about version 23.04 in our [blog post](https://www.centreon.com/centre
...
+- [UI] You can now hide the label of a resource.
+- [UX] You can now duplicate maps.
+- [Viewer] You can now define a label for a URL shape.
+- [Widget] Fixed incorrect behavior when saving the MAP widget.
+
+#### Bug fixes
+
+- [Editor] Fixed a bug where empty metric names were blocking edition.
+- [Editor] Fixed metric links when no metric name is filled in.
```
Comment (cg-tw) [inline versioned_docs/version-23.04/releases/centreon-commercial-extensions.md:37]:
```suggestion
- [Editor] Fixed metric links when no metric name was filled in.
```

File: versioned_docs/version-23.04/releases/centreon-commercial-extensions.md [hunk 14]
```diff
@@ -18,6 +18,38 @@ Read more about version 23.04 in our [blog post](https://www.centreon.com/centre
...
+- [Server] Enable to parse metric from Broker NEB packets.
```
Comment (cg-tw) [inline versioned_docs/version-23.04/releases/centreon-commercial-extensions.md:38]:
```suggestion
- [Server] Made it possible to parse metrics from Broker NEB packets.
```

File: versioned_docs/version-23.04/releases/centreon-commercial-extensions.md [hunk 15]
```diff
@@ -18,6 +18,38 @@ Read more about version 23.04 in our [blog post](https://www.centreon.com/centre
...
+- [Server] Fixed a critical error when editing a Geo view.
+- [Server] Fixed an issue causing MAP server to crash at start up due to service group loading.
+- [Server] Fixed an issue preventing to save a map with "not a diagram" error message.
```
Comment (cg-tw) [inline versioned_docs/version-23.04/releases/centreon-commercial-extensions.md:41]:
```suggestion
- [Server] Fixed an issue preventing users from saving a map with a "not a diagram" error message.
```

File: versioned_docs/version-23.04/releases/centreon-commercial-extensions.md [hunk 16]
```diff
@@ -18,6 +18,38 @@ Read more about version 23.04 in our [blog post](https://www.centreon.com/centre
...
+- [Server] Fixed an issue which made diagnostic to raise an error if a custom URI is used.
```
Comment (cg-tw) [inline versioned_docs/version-23.04/releases/centreon-commercial-extensions.md:42]:
```suggestion
- [Server] Fixed an issue which made diagnostic.sh raise an error if a custom URI was used.
```

File: versioned_docs/version-23.04/releases/centreon-commercial-extensions.md [hunk 17]
```diff
@@ -18,6 +18,38 @@ Read more about version 23.04 in our [blog post](https://www.centreon.com/centre
...
+- [UI] Fixed cosmetic issues on breadcrumb chips.
+- [UI] Fixed the selection of layers in Geo views.
+- [UI] Fixed view freezes beyond 500 elements.
+- [Viewer] Fixed a regression causing output not to be displayed in widget output.
```
Comment (cg-tw) [inline versioned_docs/version-23.04/releases/centreon-commercial-extensions.md:46]:
```suggestion
- [Viewer] Fixed a regression causing output not to be displayed in the output widget.
```

File: versioned_docs/version-23.04/releases/centreon-commercial-extensions.md [hunk 18]
```diff
@@ -18,6 +18,38 @@ Read more about version 23.04 in our [blog post](https://www.centreon.com/centre
...
+- [Viewer] Fixed an issue making links not to be reactivated when the associated resource goes back on line.
```
Comment (cg-tw) [inline versioned_docs/version-23.04/releases/centreon-commercial-extensions.md:47]:
```suggestion
- [Viewer] Fixed an issue that caused links not to be reactivated when the associated resource went back online.
```

File: versioned_docs/version-23.04/releases/centreon-commercial-extensions.md [hunk 19]
```diff
@@ -26,6 +58,20 @@ Release date: `April 26, 2023`
 
 ## Centreon BAM
 
+### 23.04.2
+
+Release date: `July 20, 2023`
+
+#### Enhancements
+
+- [Rules] Fixed boolean rules when resource contains boolean operator in its name.
```
Comment (cg-tw) [inline versioned_docs/version-23.04/releases/centreon-commercial-extensions.md:67]:
```suggestion
- [Rules] Fixed boolean rules when resource contained a boolean operator in its name.
```

File: versioned_docs/version-23.04/releases/centreon-commercial-extensions.md [hunk 20]
```diff
@@ -26,6 +58,20 @@ Release date: `April 26, 2023`
...
+
+#### Bug fixes
+
+- [Rules] Fixed boolean rules when resource contains boolean operator in its name.
```
Comment (cg-tw) [inline versioned_docs/version-23.04/releases/centreon-commercial-extensions.md:71]: in enhancements

File: i18n/fr/docusaurus-plugin-content-docs/version-23.04/releases/centreon-commercial-extensions.md [hunk 21]
```diff
@@ -46,6 +92,21 @@ Release date: `April 26, 2023`
...
+
+Release date: `July 20, 2023`
+
+#### Enhancements
+
+- [Licensing] Improved automatic license renewal.
+
+#### Bug fixes
+
+- [Install] Added missing Gorgone dependencies required by MBI ETL.
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs/version-23.04/releases/centreon-commercial-extensions.md:105]:
```suggestion
- [Install] Added missing Gorgone dependencies required by the MBI ETL.
```

File: i18n/fr/docusaurus-plugin-content-docs/version-23.04/releases/centreon-commercial-extensions.md [hunk 22]
```diff
@@ -46,6 +92,21 @@ Release date: `April 26, 2023`
...
+- [Reporting] Fixed the display of ongoing alert duration in Hostgroups-Service-Current-Events report.
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs/version-23.04/releases/centreon-commercial-extensions.md:106]:
```suggestion
- [Reporting] Fixed the display of the duration of ongoing alerts in the Hostgroups-Service-Current-Events report.
```

File: versioned_docs/version-23.04/releases/centreon-commercial-extensions.md [hunk 23]
```diff
@@ -26,6 +58,20 @@ Release date: `April 26, 2023`
...
+- [UI] Fixed the column name display in the acknowledgment description pop-up.
+- Fixed retrieving list of BAs for users with ACL.
```
Comment (cg-tw) [inline versioned_docs/version-23.04/releases/centreon-commercial-extensions.md:73]:
```suggestion
- Fixed how lists of BAs were retrieved for users with ACLs.
```

File: versioned_docs/version-23.04/releases/centreon-commercial-extensions.md [hunk 24]
```diff
@@ -47,6 +93,21 @@ Release date: `April 26, 2023`
...
+
+Release date: `July 20, 2023`
+
+#### Enhancements
+
+- [Licensing] Improved automatic license renewal.
+
+#### Bug fixes
+
+- [Install] Added missing Gorgone dependencies required by MBI ETL.
```
Comment (cg-tw) [inline versioned_docs/version-23.04/releases/centreon-commercial-extensions.md:106]:
```suggestion
- [Install] Added missing Gorgone dependencies required by the MBI ETL.
```

File: versioned_docs/version-23.04/releases/centreon-commercial-extensions.md [hunk 25]
```diff
@@ -47,6 +93,21 @@ Release date: `April 26, 2023`
...
+- [Reporting] Fixed the display of ongoing alert duration in Hostgroups-Service-Current-Events report.
```
Comment (cg-tw) [inline versioned_docs/version-23.04/releases/centreon-commercial-extensions.md:107]:
```suggestion
- [Reporting] Fixed the display of the duration of ongoing alerts in the Hostgroups-Service-Current-Events report.
```

File: versioned_docs/version-23.04/releases/centreon-commercial-extensions.md [hunk 26]
```diff
@@ -47,6 +93,21 @@ Release date: `April 26, 2023`
...
+- [UI] Added CSS in global style files to fix the MBI widget visibility.
+- [UI] Fixed empty "Current usage" column in mbi-storage-list-near-saturation widget.
```
Comment (cg-tw) [inline versioned_docs/version-23.04/releases/centreon-commercial-extensions.md:109]:
```suggestion
- [UI] Fixed empty "Current usage" column in the mbi-storage-list-near-saturation widget.
```

**Review by cg-tw — CHANGES_REQUESTED** [review]

**Review by cg-tw — APPROVED** [review]

**Review by nmagniezCentreon — APPROVED** [review]

**Pull request merged**